crypto/evp/evp_enc.c

```diff
@@ -336,7 +341,10 @@ int EVP_EncryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl,
         }
     }
     i = ctx->buf_len;
-    OPENSSL_assert(bl <= (int)sizeof(ctx->buf));
+    if (!ossl_assert(bl <= (int)sizeof(ctx->buf))) {
+        EVPerr(EVP_F_EVP_ENCRYPTUPDATE, ERR_R_INTERNAL_ERROR);
+        return 0;
+    }
```

Comment:
Well, as well as absolute majority of later similar occurrences... Let's say, I'll comment if I find non-similar case...